Products/PloneKeywordManager/testing.py

```diff
@@ -8,6 +8,15 @@
 from plone.testing import z2
 
 
+try:
```

Comment:
use the API: http://docs.plone.org/develop/plone.api/docs/env.html#plone-version